plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/SSLCetrificatesPreference.java

```diff
@@ -0,0 +1,173 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.preferences;
+
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.jboss.tools.openshift.common.core.utils.X509CertificateParser;
+import org.jboss.tools.openshift.internal.ui.OpenShiftUIActivator;
+
+public class SSLCetrificatesPreference {
```

Comment:
Ok, done
(see http://www.ecenglish.com/learnenglish/lessons/can-you-read,
e.g. "Yaeh and I awlyas tghhuot slelinpg was ipmorantt!" :) )